packages/jni-swig-stub/src/main/jni/realm_api_helpers.h

```diff
@@ -137,4 +137,16 @@ realm_sync_thread_error(realm_userdata_t userdata, const char* error);
 realm_scheduler_t*
 realm_create_generic_scheduler();
 
+realm_sync_socket_t* realm_sync_websocket_new(int64_t sync_client_config_ptr, jobject websocket_transport);
```

Comment:
We're already using `int64_t` for pointers. What would be the difference? anyway it will hold the value of `jlong`  which should be fine (even on 32 bits arch I believe)